packages/mdx-components/src/main/color-block/color-block.stories.jsx

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
I think we talked about this, but what was the purpose of this SB? 😅. I think there's a lot this one's missing compared to the one we have on the web-app:
<img width="898" alt="image" src="https://user-images.githubusercontent.com/40550942/183942248-01ecb3cc-f23b-451d-a15a-3b2bce272cc0.png">
VS
<img width="1561" alt="image" src="https://user-images.githubusercontent.com/40550942/183942322-90700d6e-a329-4c06-8c35-3b84a2b9a090.png">

Reply:
Fair question! A few purposes:

1. New home for our existing stories that apply to mdx components
2. Aids in mdx-component development (no need to have the web-app up and running)
3. Enables eventual snapshot testing specifically for the mdx-components to prevent regression

Reply:
Updated to match Alison's recommendations. Info now has parity with the one in the web-app.